src/Stat.ts

```diff
@@ -0,0 +1,84 @@
+export const statDict: {[Stat: string]: number} = {
```

Comment:
I think I would implement this as a data class that has a field for each of these rather than as a dictionary.